modules/mobile/spec/support/vcr_cassettes/appointments/get_mfs_facilities.yml

```diff
@@ -0,0 +1,262 @@
+---
+http_interactions:
```

Comment:
Is there a reason this vcr contains 3 requests to 3 different endpoints? In general it seems like we only do one request per cassette, which makes sense to me because it ensures that you know which request is being used. I could see maybe combining requests into a single file if they're closely coupled (i.e., MFS needs to make multiple calls that are always the same) but I don't think that's what's happening here because two of the calls return 403, which doesn't sound like a success case.

Reply:
It doesn't work without the other 2 endpoints.. (Maybe its to receive appointment data since the mfs endpoint is just facilities?). But I believe the 403 return was due to an issue found with the format of the facility id during the time of recording (when has been fixed)

Reply:
Interesting. Thanks for explaining. It's definitely a little atypical and I'd have to do some exploring to understand how those 403s aren't problematic, but I guess they're not.